test/amp-caching/amp-caching-test.js

```diff
@@ -0,0 +1,168 @@
+import { buildSW } from '../../index';
+import { promisify } from 'util';
+import { writeFile, unlink } from 'fs';
+import { join } from 'path';
+
+describe('AMP Caching Module', function() {
+  const driver = global.__AMPSW.driver;
+  const serviceWorkerPath = join('test', 'amp-caching-sw.js');
+  this.timeout(5000);
+
+  before(async () => {
+    const generatedSW = await buildSW();
+    const promisifiedWriteFile = promisify(writeFile);
+    await promisifiedWriteFile(serviceWorkerPath, generatedSW);
+    await driver.get('http://localhost:6881/test/index.html');
+  });
+
+  after(async () => {
+    const promisifiedUnlink = promisify(unlink);
```

Comment:
I'd recommend just hoisting this to the top level, right after your `imports`.

```javascript
import * as fs from 'fs';
import { promisify } from 'util';

const unlink = promisify(fs.unlink);
```

Reply:
done